packages/@ember/object/tests/es-compatibility-test.js

```diff
@@ -275,83 +251,83 @@ moduleFor(
       SubEmberObject.metaForProperty('foo');
     }
 
-    // TODO: Revisit this
-    '@test observes / removeObserver on / removeListener interop'(assert) {
-      let fooDidChangeBase = 0;
-      let fooDidChangeA = 0;
-      let fooDidChangeB = 0;
-      let someEventBase = 0;
-      let someEventA = 0;
-      let someEventB = 0;
-      class A extends EmberObject.extend({
-        fooDidChange: observer('foo', function () {
-          fooDidChangeBase++;
-        }),
-
-        onSomeEvent() {
-          someEventBase++;
-        },
-      }) {
-        init() {
-          super.init();
-          this.foo = 'bar';
-        }
-
-        fooDidChange() {
-          super.fooDidChange();
-          fooDidChangeA++;
-        }
-
-        onSomeEvent() {
-          super.onSomeEvent();
-          someEventA++;
-        }
-      }
-
-      class B extends A {
-        fooDidChange() {
-          super.fooDidChange();
-          fooDidChangeB++;
-        }
-
-        onSomeEvent() {
-          super.onSomeEvent();
-          someEventB++;
-        }
-      }
-
-      removeObserver(B.prototype, 'foo', null, 'fooDidChange');
-      removeListener(B.prototype, 'someEvent', null, 'onSomeEvent');
-
-      assert.equal(fooDidChangeBase, 0);
-      assert.equal(fooDidChangeA, 0);
-      assert.equal(fooDidChangeB, 0);
-
-      assert.equal(someEventBase, 0);
-      assert.equal(someEventA, 0);
-      assert.equal(someEventB, 0);
-
-      let a = A.create();
-      a.set('foo', 'something');
-
-      // TODO: Generator transpilation code doesn't play nice with class definitions/hoisting
-      return runLoopSettled().then(async () => {
-        assert.equal(fooDidChangeBase, 1);
-        assert.equal(fooDidChangeA, 1);
-        assert.equal(fooDidChangeB, 0);
-
-        let b = B.create();
-        b.set('foo', 'something');
-        await runLoopSettled();
-
-        assert.equal(fooDidChangeBase, 1);
-        assert.equal(fooDidChangeA, 1);
-        assert.equal(fooDidChangeB, 0);
-
-        a.destroy();
-        b.destroy();
-      });
-    }
+    // TODO: Determine if there's anything useful to test here with observer helper gone
```

Comment:
We need to decide if this, and subsequent tests, should be updated and kept around.